docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[ml-dataframes]]
+=== {dataframes-cap}
+
+{dataframes-cap} feature is available in 7.2 and later.
+
+A _{dataframe}_ is a transformation of a dataset by certain rules defined during
+the creation of the {dataframe}. You can think of it like a spreadsheet or a 
+data table that makes your data ready to be analyzed and organized.
+
+{es} datasets consist of individual documents that have fields and
```

Comment:
it _depends_ ;-) This sounds a bit to general for me, you can also upload summarized data into elasticsearch or you can summarize data using external tools (e.g. hadoop/spark). Maybe you can say:

A lot of datasets are organized as a stream of events: Each event is a individual document, e.g. a single item purchase, data frame pivots allow you to summarize this data, bringing it into a an organized, more analysis friendly format, e.g. summarizing all purchases of a single customer...

Reply:
Thank you for your suggestion, I amended the text accordingly.